src/plugins/dataflow/nodes/controls/sensor-value-control.tsx

```diff
@@ -26,10 +26,14 @@ export class SensorValueControl extends Rete.Control {
     this.updateUnits();
 
     this.component = (compProps: { value: number; units: string; }) => {
-      // Only display up to 3 decimal places
-      const places = 1000;
-      const displayValue = isNaN(compProps.value) ? kEmptyValueString
-        : Math.round(compProps.value * places) / places;
+      // If decimal places are specifed for this sensor type, use them, otherwise default to 0
+      const sensorType = NodeSensorTypes.find((s: any) => s.type === this.node.data.type);
+      const decimalPlaces = sensorType?.decimalPlaces ? sensorType.decimalPlaces : 0;
```

Comment:
Per @tealefristoe 's comment in the discussion, did you want to make the default 2 decimal places?

Reply:
Thanks for noticing that, made the change